src/sql/workbench/services/query/common/dataService.ts

```diff
@@ -135,9 +135,6 @@ export class DataService {
 
 	/**
 	 * send request to save the selected result set as csv
```

Comment:
shouldn't this have saveRequest as a param?

Reply:
This only checks that if a param is included in the JS Doc that it exists in the signature. To detect missing params requires the `require-param` rule. But there's hundreds of those so I'm going to follow up with that as a separate PR.

Reply:
oh, I just saw your last sentence around this being a follow-up task 😄

Reply:
I could have added this one here, which I did do for some others. But I realized part way through that it would just be a waste of effort - especially since the `require-param` rule has an auto-fix that should make that one a very simple thing to do so doing it now was just extra effort.